repositories/log_blacklist_repository.py

```
@@ -63,8 +63,8 @@ async def is_entity_blacklisted(self, guild_id: str, entity_id: str, blacklist_t
         table = blacklist_type.table
         column = blacklist_type.column
         query = f"SELECT {column} FROM {table} WHERE guild_id = %s AND {column} = %s"
-        result = await execute_query(query, (guild_id, entity_id))
-        return result[0] if result else None
+        result: tuple[Any, ...] | None = await execute_query(query, (guild_id, entity_id))
```

Flake8 (7.3.0)

[error] undefined name 'Any'

(F821)